src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private final int maxFigures = 5;

```

Comment:
The `maxFigures` variable should be declared as a constant. In Java, constants are declared using the `final` keyword and are usually written in upper case. For example, `private static final int MAX_FIGURES = 5;`.